src/latexraw.jl

```diff
@@ -131,26 +139,30 @@ function _latexraw(i::Number; fmt=PlainNumberFormatter(), kwargs...)
     return fmt(i)
 end
 
-function _latexraw(i::Char; convert_unicode=true, kwargs...)
+_latexraw(i::Char; convert_unicode=true, kwargs...) =
     LaTeXString(convert_unicode ? unicode2latex(string(i)) : string(i))
-end
 
 function _latexraw(i::Symbol; convert_unicode=true, snakecase=false, safescripts=false, kwargs...)
-    str = string(i == :Inf ? :∞ : i)
+    str = string(i === :Inf ? :∞ : i)
     str = convert_subscript(str; snakecase=snakecase)
     convert_unicode && (str = unicode2latex(str; safescripts=safescripts))
     return LaTeXString(str)
 end
 
-function _latexraw(i::String; kwargs...)
+_latexraw(i::String; parse=true, kwargs...) = _latexraw(Val(parse), i; kwargs...)
```

Comment:
Any strong motivation to go to a `Val{Bool}` here? Seems like this could just be a single `if` with an early return, which would leave the code more self-explanatory and so easier to maintain.

Reply:
Well I always prefer to dispatch on types, especially since the `_latexraw(Val{true}, ...)` uses `try catch` which makes the function unpure (side effects).